gateway-sp-comms/src/lib.rs

```diff
@@ -4,6 +4,10 @@
 
 // Copyright 2022 Oxide Computer Company
 
+// Required nightly features for `usdt`
+#![cfg_attr(target_os = "macos", feature(asm_sym))]
+#![feature(asm)]
```

Comment:
I thought that feature(asm) was no longer needed?

Reply:
It appears to be required still; taking it out gives

```
error[E0658]: use of unstable library feature 'asm': inline assembly is not stable enough for use and is subject to change
   --> gateway-sp-comms/src/recv_handler/mod.rs:210:1
    |
185 | /         probes::recv_serial_console!(|| (
186 | |             &port,
187 | |             &packet.component,
188 | |             packet.offset,
189 | |             packet.data.as_ptr() as usize as u64,
190 | |             u64::from(packet.len)
191 | |         ));
    | |__________- in this macro invocation
...
210 |   #[usdt::provider(provider = "gateway_sp_comms")]
    |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

for me on macOS.